packages/augur-state/src/db/DB.test.ts

```diff
@@ -1,103 +1,25 @@
 import { TrackedUsers } from "./TrackedUsers";
 import { DB, UserSpecificEvent } from "./DB";
-import { EthersProvider } from "ethers-provider";
+import { EthersWeb3Provider } from "@augurproject/ethers-provider";
```

Comment:
This line will probably be different as it's based on temporary code I used while waiting for the proper implementation to be completed. I'm not sure which ticket that has though. @adrake33?

Reply:
Yeah, this line should be `import { EthersProvider, Web3AsyncSendable } from "ethers-provider";`